docs/src/user_docs/assembly/code_organization.md

```diff
@@ -105,62 +112,55 @@ end
+3. We can import the `baz` item directly into scope with `use foo::bar::baz`, allowing us to treat it like a locally-defined item, e.g. `exec.baz`
+
+All three are demonstrated below:
 
 ```
-use.std::math::u64
+use foo::bar
+use foo::bar::baz
 
 begin
-  ...
+  exec.::foo::bar::baz # no import, fully-qualified path
+  exec.bar::baz # uses the first import, partially-qualified path
+  exec.baz # uses the second import, unqualified path
 end
 ```
 
-In this example, the `std::math::u64` module is imported as `u64`, the default "alias" for the imported module. You can specify a different alias like so:
+Let's say we have a local symbol that conflicts with the symbol we're trying to import, we can work around this by renaming the imported symbol to avoid the conflict, as shown below:
 
 ```
-use.std::math::u64->bigint
+use foo::bar->bar2
 ```
 
-This would alias the imported module as `bigint` rather than `u64`. The alias is needed to reference items from the imported module, as shown below:
-
-```
-use.std::math::u64
+This would bring `bar` into scope as `bar2` instead:
 
-begin
-    push.1.0
-    push.2.0
-    exec.u64::wrapping_add
-end
 ```
+use foo::bar->bar2
 
-You can also bypass imports entirely, and specify an absolute procedure path, which requires prefixing the path with `::`. For example:
-
-```
 begin
-  push.1.0
-  push.2.0
-  exec.::std::math::u64::wrapping_add
+    exec.bar2::baz
 end
 ```
 
-In the examples above, we have been referencing the `std::math::u64` module, which is a module in the [Miden Standard Library](../stdlib/index.md). There are a number of useful modules there, that provide a variety of helpful functionality out of the box.
+If the assembler cannot resolve external symbol references to a known module or item, assembly will fail. You can register modules with the assembler when instantiating it, either in source form, or precompiled form. See the [miden-assembly docs](https://crates.io/crates/miden-assembly) for details. The assembler will use this information to resolve references to imported items during assembly.
 
-If the assembler does not know about the imported modules, assembly will fail. You can register modules with the assembler when instantiating it, either in source form, or precompiled form. See the [miden-assembly docs](https://crates.io/crates/miden-assembly) for details. The assembler will use this information to resolve references to imported procedures during assembly.
-
-#### Re-exporting procedures
-A procedure defined in one module can be re-exported from a different module under the same or a different name. For example:
+#### Re-exporting items
```

Comment:
Should we explicitly mention somewhere in this section that we cannot attach attributes to re-exports?